data/contributors.json

```
@@ -0,0 +1,33 @@
+{
+    "leads": [
+        {
+          "name": "LaShawn Toyoda",
+          "githubHandle": "theyokohamalife",
+          "twitterHandle": "theyokohamalife",
+          "role": "Project Originator and Lead Developer"
+        },
+        {
+          "name": "Ann Kilzer",
+          "githubHandle": "ann-kilzer",
+          "twitterHandle": "stonecoldkilzer",
+          "role": "Dev, QA, and localization design",
```

```suggestion
          "role": "Dev, tooling, and localization design",
```

This is a little closer to my role, as if I were *really doing QA* I'd insist on adding a testing framework 😆 

No time 😆

---

sure =). Any adjustments for anyone else?

---

haha. I'd say that's SDET rather than QA but now we're splitting hairs =). I'll change it